pootle/apps/pootle_store/forms.py

```diff
@@ -280,11 +284,16 @@ def clean_target_f(self):
 
             return value
 
-        def clean_state(self):
+        def _clean_state(self):
+            # Handle clean_state here because it depends on clean_suggestion.
```

Comment:
hmm - im a bit uncomfortable calling it `_clean_state` as it is confusing relative to what `clean_state` would do.

As there is only this logic, i would keep in the clean method.